pkg/kubernetes/namespace/injected.go

```diff
@@ -0,0 +1,55 @@
+package namespace
+
+import (
+	"fmt"
+
+	kubernetes2 "github.com/openservicemesh/osm-health/pkg/kubernetes"
+
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openservicemesh/osm-health/pkg/common"
+	"github.com/openservicemesh/osm/pkg/constants"
+)
+
+const enabled = "enabled"
+
+// SidecarInjectionCheck implements common.Runnable
+type SidecarInjectionCheck struct {
+	client    kubernetes.Interface
+	namespace kubernetes2.Namespace
+}
+
+// IsInjectEnabled checks whether a namespace is enabled for sidecar injection.
+func IsInjectEnabled(client kubernetes.Interface, namespace kubernetes2.Namespace) common.Runnable {
+	return isInjectEnabled(client, namespace)
+}
+
+func isInjectEnabled(client kubernetes.Interface, namespace kubernetes2.Namespace) SidecarInjectionCheck {
+	return SidecarInjectionCheck{
+		client:    client,
+		namespace: namespace,
+	}
+}
+
+// Info implements common.Runnable
+func (ii SidecarInjectionCheck) Info() string {
```

Comment:
what does the name `ii` signify?

Reply:
Good question. I renamed all these receivers from `ii` to `check`.